### ISSUE TITLE
CloudSQL in version 0.9.8.3

### ISSUE THREAD

**Issue by RideLikeTheWind** [catfan/Medoo#286]
Just a note - CloudSQL (GAE) support was lost(?) in the most recent version I downloaded from your site - Add the following code below line 80 to fix:

```
case 'cloudsql':
  $dsn = 'mysql:unix_socket=/cloudsql/' . $this->server . ';dbname=' . $this->database_name;
  $commands[] = 'SET SQL_MODE=ANSI_QUOTES';
  $commands[] = $set_charset;
break;
```

Thanks for a fantastic framework!

**Comment by catfan** [comment]
Thanks for that suggestion.

As for other Cloud based SQL service, like CloudSQL from Google and Amazon RDS or more from other company, it is hard to support all.

But it is should be considered to be configurable for supporting potential database.

**Comment by RideLikeTheWind** [comment]
I agree - I think this was a supported service in a previous version, which is where I lifted the above code. I will look and see if I can create a way to provide other services within this framework without the need for inbuilt service allocation. 

Cheers!

**Comment by catfan** [comment]
It will be available on next version! 524ce57ec3dc07f86499f139e6e1918d80037578
